code/game/objects/items/weapons/gift_wrappaper.dm

```diff
@@ -357,7 +357,7 @@
 		/obj/item/device/megaphone,
 		/obj/item/device/violin)
 
-	var/atom/movable/I = new gift_type(M)
+	var/atom/movable/I = new gift_type(get_turf(M))
```

Comment:
This one already handles put-in-hands failure, no?

Reply:
Nope, it seems to cause things like animals and random items to be stuck into your mob contents.